src/main/groovy/com/avioconsulting/mule/linter/model/Application.groovy

```diff
@@ -61,6 +62,42 @@ class Application {
         }
     }
 
+    List<MuleComponent> findComponents(String componentType, String namespace) {
+        List<MuleComponent> comps = []
+        configurationFiles.each { configFile ->
+            comps += configFile.findComponents(componentType, namespace)
+        }
+        return comps
+    }
+
+    List<FlowComponent> getFlows() {
+        List<FlowComponent> comps = []
+        configurationFiles.each { configFile ->
```

Comment:
`return configurationFiles.collect { it.flows }.flatten() `

Reply:
done.